GWAS/PCA.ipynb

```diff
@@ -642,7 +742,7 @@
     "        mu = t(t(mu_pc) - m)\n",
     "        pc$mahal = rowSums(mu %*% robust_inv(s) * mu)\n",
     "        pc$p <- pchisq(pc$mahal, df=nrow(s), lower.tail=FALSE)\n",
-    "        manh_dis_sq_cutoff = quantile(pc$mahal, probs=prob)\n",
+    "        manh_dis_sq_cutoff = quantile(pc$mahal, probs=prob,na.rm=TRUE)\n",
```

Comment:
why do we have NA? I think that's a problem! I dont expect NA in the distances computed. We cannot just ignore it. We need to see what's going on.